packages/typespec-vscode/package.json

```diff
@@ -98,6 +98,15 @@
               "additionalProperties": false
             }
           },
+          "typespec.lsp.emit": {
+            "type": "array",
+            "default": null,
+            "description": "List of emitters that should be included in compilation in typespec LSP server when supporting LSP related functionalities.\n\nExample:\n\"typespec.lsp.emit\": [\n  \"@typespec/openapi3\"\n]",
```

Comment:
Should describe that this will just run the emitter in dry mode for emitters that support it even if you enable it